crates/atiny-fs/src/lib.rs

```diff
@@ -160,7 +160,7 @@ impl<'r, T: VirtualFile> VirtualFileSystem<T, io::Error> for FileSystem<'r, T> {
         self.read_file(path, parent)
     }
 
-    fn read_file(&mut self, path: &str, parent: Option<NodeId>) -> io::Result<File> {
```

Comment:
Why change everything to String?

Reply:
Cause i needed it to be mutable in `lowercase_first_letter`, so i thought it was better to just own it from the beggining instead of cloning into an owned value later